src/vaibhav/GetAllDigitFromString.java

```diff
@@ -0,0 +1,36 @@
+package vaibhav;
+
+/*program 2 : sum of all numbers in a give string
+input: te123ch9kj8
+output : 123 + 9 + 8 = 140*/
+
+
+public class GetAllDigitFromString {
+	
+	void getAllDigit(String str) {
+		int sum = 0;
+		String temp = "";
+		boolean flag = false;
+		for (int index = 0 ; index < str.length();index++) {
+			char ch=str.charAt(index);
+			if(Character.isDigit(ch)) {
+				flag = true;
+				temp = temp+Character.getNumericValue(ch);				
+			}
+			if (Character.isLetter(ch) && flag == true  || index == str.length()-1 )
```

Comment:
neven put true and == make no sense, as Character.isLetter(ch) && flag have boolean values